### DISCUSSION TITLE
Budget overview for recurring expenses

### DISCUSSION THREAD

**Discussion by kevinrmccoy** [voxeltycoon/issues#799] [UI&UX]
It'd be great if there was a subtotal in the budget overview that let me see if my company is making or losing money through normal operations, without taking into account money spent on construction or taking/repaying loans (and maybe research?  I'm not sure which side of the subtotal that would fit best on.)  It's easier to make spending decisions if you have a sense of what your net cashflow is without those expenses.

**Comment by WoosterUK** [comment]
Yes! Generally, better accounting standards would help. One of my main bugbears with TTD and its direct descendants is that you don't have proper asset and liability accounts, so seeing where you are is really difficult. Compare that with Railroad Tycoon 2, for example, which stands of course in the line of descent from TTD though not so directly: it has a balance sheet, profit and loss accounts, the works. It gives the game much more richness in terms of what it can do with the financial side of running the business.
